src/pages/[platform]/build-a-backend/add-aws-services/geo/existing-resources/index.mdx

```diff
@@ -0,0 +1,207 @@
+        default: '<your-default-index>' // REQUIRED - Amazon Location Service Place Index name to set as default
+      },
+      geofenceCollections: {
+        items: ['<your-geo-collection>'], // REQUIRED - Amazon Location Service Geofence Collection name
+        default: '<your-default-collection>' // REQUIRED - Amazon Location Service Geofence Collection name to set as default
+      },
+      region: '<your-geo-region>' // REQUIRED - Amazon Location Service Region
+    }
+  }
+});
+```
+
+Now you can proceed to [displaying a map](/[platform]/build-a-backend/add-aws-services/geo/maps/) or [adding location search](/[platform]/build-a-backend/add-aws-services/geo/location-search/) to your app.
+</InlineFilter>
+
+
+<InlineFilter filters={['android']}>
+## In your app configuration
+
+Amplify Geo is dependent on your Amplify Auth category. If it is not already configured, then you will have to manually configure it as well. In order to manually configure Amplify Geo category, you must edit `amplifyconfiguration.json` in your project's `src/main/res/raw` directory with your information from existing Amazon Cognito and Amazon Location Service resources.
+
+```json
+{
+    "Version": "1.0",
+    "geo": {
+        "plugins": {
+            "awsLocationGeoPlugin": {
+                "region": <your-geo-region>,
+                "maps": {
+                    "items": {
+                        <your-map-name>: {
+                            "style": <your-map-style>
+                        }
+                    },
+                    "default":<your-preferred-default-map>
+                },
+                "searchIndices": {
+                    "items": [
+                        <your-geo-index>
+                    ],
+                    "default": <your-default-index>
+                }
+            }
+        }
+    },
+    "auth": {
+        "plugins": {
+            "awsCognitoAuthPlugin": {
+                "Version": "0.1.0",
+                "IdentityManager": {
+                    "Default": {}
+                },
+                "CredentialsProvider": {
+                    "CognitoIdentity": {
+                        "Default": {
+                            "PoolId": <your-identity-pool-id>,
+                            "Region": <your-identity-pool-region>
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+Now you can proceed to [displaying a map](/gen1/[platform]/build-a-backend/more-features/geo/maps/) or [adding location search](/gen1/[platform]/build-a-backend/more-features/geo/location-search/) to your app.
+</InlineFilter>
+
+
+<InlineFilter filters={['swift']}>
+## In your app configuration
+
+Amplify Geo is dependent on your Amplify Auth category. If it is not already configured, then you will need to [configure it](/gen1/[platform]/build-a-backend/auth/existing-resources) as well.
+
```

Comment:
Just wanted to verify, should this link be pointing to gen1?

Reply:
updated the links